components/src/organisms/quantity-update/QuantityUpdate.stories.js

```diff
@@ -0,0 +1,95 @@
+import { storiesOf } from '@storybook/vue'
+import { action } from '@storybook/addon-actions'
+import { number, text } from '@storybook/addon-knobs'
+
+import AQuantityUpdate from './QuantityUpdate.vue'
+
+const info = 'Check **Knobs** tab to edit component properties dynamically.'
+
+storiesOf('Organisms/Quantity Update', module)
+  .addParameters({ info })
+  .add('Default', () => ({
+    components: { AQuantityUpdate },
+    props: {
+      labelKnob: {
+        default: text('Label', 'Quantity')
+      },
+      minQtyKnob: {
+        default: number('Min qty', 0)
+      },
+      maxQtyKnob: {
+        default: number('Max qty', 100)
+      }
+    },
+    methods: {
+      updateVal: action('Quantity')
+    },
+    template: `
+      <a-quantity-update
+        :value="1"
+        @update="updateVal"
+        :label="labelKnob"
+        input-id="qty-update"
+        :min-qty="minQtyKnob"
+        :max-qty="maxQtyKnob"
+        decrement-aria-label="Decrease the quantity"
+        decrement-icon-title="Minus mark"
+        increment-aria-label="Increase the quantity"
+        increment-icon-title="Plus mark"
```

Comment:
What about v-model? Shouldn't we support this?

Reply:
Right! Done :D